package.json

```diff
@@ -28,10 +28,7 @@
     "jest-environment-jsdom": "^29.3.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-<<<<<<< HEAD
```

Comment:
I think I have this issue but is it possible to get rid of these head and numbers?